DOCS-6078 remove links from headings (#882)
clerk/clerk-docs: docs/organizations/metadata.mdx

# Organization metadata

> Organization metadata allows you to store information about an organization that is not part of the standard fields. This can be useful for storing custom attributes that are specific to your application.

Reviewer: ```suggestion
Organization metadata allows you to store information about an organization that is not part of the standard fields, such as custom attributes that are specific to your application.
```

There are two types of metadata: **public** and **private**.

Both public and private metadata are set and can be accessed from the [Backend API](https://clerk.com/docs/reference/backend-api). Only public metadata can be accessed from the [Frontend API](https://clerk.com/docs/reference/frontend-api/).

> Both the `Organization` and `Organization Membership` objects have the metadata fields: `publicMetadata` and `privateMetadata`.

Reviewer: ```suggestion
Both the `Organization` and `Organization Membership` objects have the following metadata fields:
- `publicMetadata` –  Useful for setting metadata from your backend and have them displayed as read-only on the frontend.
- `privateMetadata` – Useful for setting custom attributes that contain sensitive information that should not be displayed on the frontend.
```

Combines this and the following lines for less repitition. Also, what do we mean by "displayed" here? Is `publicMetadata` accessible on the `Organization` object in the frontend, but `privateMetadata` is not? Or is it just expected that the developer won't display `privateMetadata` despite having access to it in the frontend?

The `publicMetadata` property should be used if you need to set some metadata from your backend and have them displayed as read-only on the frontend.

The `privateMetadata` property should be used if custom attributes contain sensitive information that should not be displayed on the frontend.

## Set organization metadata

To ease the flow of setting metadata, Clerk provides the `updateOrganizationMetadata()` and `updateOrganizationMembershipMetadata()` methods from the [Backend SDK](https://clerk.com/docs/reference/backend-api), which is a wrapper around the [Backend API](https://clerk.com/docs/reference/backend-api).

> <Callout type="warning">

Reviewer: I agree with moving this to where it is 👍
